app/src/main/res/values/strings.xml

```diff
@@ -300,6 +306,10 @@
 
     <string name="failed_to_check_updates">Failed to check for updates</string>
     <string name="dismiss_temporary">Not now</string>
-    <string name="update_available">New update available</string>
-    <string name="update_available_description">A new version (%s) is available for download.</string>
+    <string name="new_update_available">New update available</string>
+    <string name="new_update_available_description">A new version (%s) is available for download.</string>
```

Comment:
Why is there a `new_` version and a "regular" version of these strings? Seems confusing

Reply:
I merged changes from dev and strings started conflicting, supposedly from the new update pop-up on launch

I can rename it to update_available_dialog?

Reply:
I don't know. Try to reuse strings if possible

Reply:
I've updated the names of these strings to be consistent with other strings in the app, but wasn't able to reuse them